### PR TITLE
Feat/chart deprecated addition

### PR DESCRIPTION
**Pull request by hardik12-3**

### CODE REVIEW

**Review by rashmirai21 — APPROVED** [review]

File: src/components/charts/util/ChartSelect.tsx [hunk 1]
```diff
@@ -73,7 +74,13 @@ const ChartSelect: React.FC<ChartSelectProps> = ({ chart, selectChart, addChart,
                 <span className="chart-grid-item__title-repo">{chart.chart_name}</span>
                 <span>/{chart.name}</span>
             </div>
-            <div className="chart-grid-item__chart-version">{chart.version}</div>
+            <div className="flexbox" style={{alignItems: 'center'}}>
+                <div className="chart-grid-item__chart-version" style={{margin: '0 12px 0 0'}}>{chart.version}</div>
```
Comment (pghildiyal) [inline src/components/charts/util/ChartSelect.tsx:78]: use mr-12 from formulae.scss and similarly at other places

File: src/components/common/DeprecatedUpdateWarn.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import triangleAlert from '../../assets/icons/ic-alert-triangle.svg';
+
+export class DeprecatedWarn extends Component {
+    render() {
+        return (
+            <div className="flexbox" style={{alignItems: 'center'}}>
+                <img src={triangleAlert} className="icon-dim-16 mr-8" style={{margin: 0}}/>
```
Comment (pghildiyal) [inline src/components/common/DeprecatedUpdateWarn.tsx:8]: you are setting margin: 0, then why using mr-8 ?

File: src/components/charts/util/ChartSelect.tsx (hunk 1 above)
Comment (hardik12-3) [reply src/components/charts/util/ChartSelect.tsx:78]: Changed

File: src/components/common/DeprecatedUpdateWarn.tsx (hunk 2 above)
Comment (hardik12-3) [reply src/components/common/DeprecatedUpdateWarn.tsx:8]: removed mr-8

**Review by pghildiyal — APPROVED** [review]

**Pull request merged**